tests/robot-tests/tests/admin/bau/ancillary_files.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Resource    ../../libs/admin-common.robot
+Library  Collections
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       user signs in as bau1
+Suite Teardown    user closes the browser
+
+*** Variables ***
+${TOPIC_NAME}        %{TEST_TOPIC_NAME}
+${PUBLICATION_NAME}  UI tests - ancillary files %{RUN_IDENTIFIER}
+
+*** Test Cases ***
+Create Manage content test publication
+    [Tags]  HappyPath
+    user selects theme "Test theme" and topic "${TOPIC_NAME}" from the admin dashboard
+    user waits until page contains element    xpath://a[text()="Create new publication"]     60
+    user clicks link  Create new publication
+    user creates publication    ${PUBLICATION_NAME}
+
+Verify Manage content test publication is created
+    [Tags]  HappyPath
+    user checks page contains accordion  ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    user checks accordion section contains text  ${PUBLICATION_NAME}    Methodology
+    user checks accordion section contains text  ${PUBLICATION_NAME}    Releases
+
+Create release
+    [Tags]  HappyPath
+    user clicks element  css:[data-testid="Create new release link for ${PUBLICATION_NAME}"]
```

Comment:
Can use `user clicks testid element  Create new release link for ${PUBLICATION_NAME}`